telcell/pipeline.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+from typing import Any, Iterable, Mapping, Sequence, Tuple, Union
+
+import lir.plotting
+import numpy as np
+
+from telcell.data.models import Track, is_colocated
+from telcell.models import Model
+
+
+def run_pipeline(
```

Comment:
Misschien niet in dit PR, maar mogelijk moeten we bij deze functie opletten waar de `lrbenchmark` kunnen / moeten gaan gebruiken ipv delen van wielen opnieuw uitvinden.

Reply:
Eens, het was mij nog niet helemaal duidelijk hoe `lrbenchmark` en deze pipeline zich tot elkaar zouden kunnen verhouden. Maar inderdaad, goed mogelijk dat als we de dependency op `lrbenchmark` toevoegen een deel van deze code gerefactored moet worden.